### ISSUE TITLE
pinyin tone numbers

### ISSUE THREAD

**Issue by arthurtalkgoal** [lovell/limax#4]
Hi All,

As a common practice, China pinyin urls don't include the tone numbers.

We wonder if the url slug need the tone numbers, e.g. 
http://www.haopinyin.com/

And the following e.g.:
```
弄堂里的菜品赤醬

// instead of 
nong4-tang2-li3-di2-cai4-pin3-chi4-jiang4

// should be
nong-tang-li-di-cai-pin-chi-jiang

// or even be (though not readable for foreigners; and can be managed by separator parameter)
nongtanglidicaipinchijiang

```

Should we change the limax pinyin module?

This issue refer to : 
https://github.com/keystonejs/keystone/issues/1366#issuecomment-105292463

**Comment by lovell** [comment]
Thank you @arthurtalkgoal. Would a new `tone` option, defaulting to `true` to maintain backwards compatibility, provide what we need here?

```javascript
var slug = require('limax');
var redSauce = slug('弄堂里的菜品赤醬', {tone: false});
```

**Comment by arthurtalkgoal** [comment]
Hi @Lovell

That will be very good. 

Arthur CHAN

Co-founder
Talkgoal.com

**Comment by lovell** [comment]
@arthurtalkgoal Commit 08da49c available on master should provide what we need. Are you able to help test this before I publish to npm?

**Comment by arthurtalkgoal** [comment]
Hi @lovell, 

I just tried the commit. That works great!

Great job,
Arthur

**Comment by lovell** [comment]
Thanks for verifying and thank you for the original suggestion.

This has now been published to npm as v1.1.0 - https://www.npmjs.com/package/limax
